Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM node:8-alpine
 
-RUN apk add --no-cache gettext imagemagick librsvg git
```

Comment:
do we definitely not need git anymore?

Reply:
It looks like the automated Docker build ran successfully on this branch, and I can't think of a reason why it would be needed at runtime.

<img width="1100" alt="Screen Shot 2019-07-06 at 3 47 19 PM" src="https://user-images.githubusercontent.com/1487036/60760565-5dc54200-a005-11e9-9d53-e64a01b731c3.png">

Link: https://cloud.docker.com/u/shieldsio/repository/registry-1.docker.io/shieldsio/shields/timeline